crawl-ref/source/ability-type.h

```diff
@@ -82,7 +82,9 @@ enum ability_type
     ABIL_ZIN_VITALISATION,
     ABIL_ZIN_IMPRISON,
     ABIL_ZIN_SANCTUARY,
+#if TAG_MAJOR_VERSION == 34
     ABIL_ZIN_CURE_ALL_MUTATIONS,
```

Comment:
yeah good point. I think you can remove everything but the entry in ability_type, based on 4eb006deb9de as an example template I found; just a matter of testing save importing with a character that has full piety.